api/handlers.go

```diff
@@ -13,24 +14,39 @@ type Response struct {
 	code    int
 }
 
-// GetSnippet - Handler gets a single code snippet from service by its id
+const (
+	statusError = "error"
+	statusOK = "ok"
```

Comment:
With constants, I'm used to putting the caps. Not sure what your view on that is?